patches/minecraft/net/minecraft/network/NetHandlerPlayServer.java.patch

```diff
@@ -17,7 +17,7 @@
          if (blockpos.func_177956_o() < this.field_147367_d.func_71207_Z() - 1 || enumfacing != EnumFacing.UP && blockpos.func_177956_o() < this.field_147367_d.func_71207_Z())
          {
 -            if (this.field_184362_y == null && this.field_147369_b.func_70092_e((double)blockpos.func_177958_n() + 0.5D, (double)blockpos.func_177956_o() + 0.5D, (double)blockpos.func_177952_p() + 0.5D) < 64.0D && !this.field_147367_d.func_175579_a(worldserver, blockpos, this.field_147369_b) && worldserver.func_175723_af().func_177746_a(blockpos))
-+            double dist = field_147369_b.field_71134_c.getBlockReachDistance() + 3;
++            double dist = field_147369_b.func_110148_a(EntityPlayer.REACH_DISTANCE).func_111126_e() + 3;
```

Comment:
Can't this one and the one above be left as ` double dist = player.interactionManager.getBlockReachDistance() + 3;` ?

Reply:
I disagree. The value is part of the actual player now, not the player's interaction manager - it shouldn't be advertised as if it were. There's no downside to getting rid of these methods - especially since you have more than get/set operations available through the attribute.

Reply:
Ok, makes sense to me.